concepts/05 Widgets/Form/01 Getting Started with Form/80 Submit the Form.md

```diff
@@ -0,0 +1,282 @@
+To submit a form, add the [Button Item](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/ButtonItem/) and set the [useSubmitBehavior](/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#useSubmitBehavior) property to **true**. The Form can be submitted to a server only when all the validation checks pass successfully. 
```

Comment:
Otherwise ... what?

Reply:
Here we have no conditions. Therefore this is the only way to submit the form, otherwise the submission fails.